src/riak_pb_messages.csv

```diff
@@ -63,6 +63,10 @@
 97,TsGetResp,riak_ts
 98,TsListKeysReq,riak_ts
 99,TsListKeysResp,riak_ts
+100,TsCoverageReq,riak_ts
+101,TsCoverageResp,riak_ts
+102,TsCoverageEntry,riak_ts
```

Comment:
102 and 103  need not be here. Only Req/Resp do. It's too late to fix it here, though.